docs(releasing): align 2 days before branch. update managers (#11333)
GoogleChrome/lighthouse: docs/releasing.md

### Cadence

We aim to release every 3 weeks. Our schedule is set as follows: Two days before the [expected Chromium branch point](https://www.chromium.org/developers/calendar) (which is every six weeks) and again exactly 3 weeks after that day. These are Tuesdays.

For example, following this schedule, we will attempt a release on these dates:

* _Sep 29 2020_ (M87)
* Oct 20 2020
* _Nov 10 2020_ (M88)
* Dec 1 2020
* ...

Italicized dates are two days before the expected Chromium branch point.

The planned ship dates are added to the internal Lighthouse calendar.

If a release is necessary outside these scheduled dates, we may choose to skip the next scheduled release.

> In general, the above release dates are when new versions will be available in npm. Within 2 weekss, it will be reflected in LR / PSI. Some 10 weeks later, it will be available in Stable Chrome.

Reviewer: ```suggestion
In general, the above release dates are when new versions will be available in npm. Within 2 weeks, it will be reflected in LR / PSI. Some 10 weeks later, it will be available in Stable Chrome.
```

1. Release mgr tells the _LH public_ Hangout chat about the new version.

> 1. V & Kayce write and publish the [/updates](https://developers.google.com/web/updates/) blog post

Reviewer: ```suggestion
1. Release mgr optionally write and publish the [/updates](https://developers.google.com/web/updates/) blog post, if appropriate
```

Author: Can we remove this? afaik we've never ever done this.

> 1. Paul writes the tweet (linking the /updates post) and sends it on [@____lighthouse](https://twitter.com/____lighthouse).

Reviewer: ```suggestion
1. Addy writes the tweet (linking the /updates post) and sends it on [@____lighthouse](https://twitter.com/____lighthouse).
```
